content/collections/api/en/taxonomy.md

```diff
@@ -1751,4 +1767,405 @@ A failed request returns a `409 Bad Request` status and an error message.
+
+### Create a group property
+
+Create a group property. Send a `POST` request to the endpoint with the required information in the body.
+
+`POST https://amplitude.com/api/2/taxonomy/group-property`
+
+{{partial:tabs tabs="cURL, HTTP"}}
+{{partial:tab name="cURL"}}
+```bash
+curl --location --request POST 'https://amplitude.com/api/2/taxonomy/group-property' \
+-u '{api_key}:{secret_key}' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'group_type=GROUP_TYPE' \
+--data-urlencode 'group_property=GROUP_PROPERTY' \
+```
+{{/partial:tab}}
+{{partial:tab name="HTTP"}}
+```bash
+POST /api/2/taxonomy/group-property HTTP/1.1
+Host: amplitude.com
+Authorization: Basic {api-key}:{secret-key} #credentials must be base64 encoded
+Content-Type: application/x-www-form-urlencoded
+Content-Length: 94
+
+group_type=GROUP_TYPE&group_property=GROUP_PROPERTY
+```
+{{/partial:tab}}
+{{/partial:tabs}}
+
+{{partial:collapse name="Example: Create a group property"}}
+This example creates the group property "Group Property 1" with the description "First group property" for the group "Group 1". The group property is a boolean type.
+
+{{partial:tabs tabs="cURL, HTTP"}}
+{{partial:tab name="cURL"}}
+```bash
+
+curl --location --request POST 'https://amplitude.com/api/2/taxonomy/group-property' \
+--header 'Authorization: Basic MTIzNDU2NzgwMDoxMjM0NTY3MDA=' \
+--data-urlencode 'group_type=Group 1' \
+--data-urlencode 'group_property=Group Property 1' \
+--data-urlencode 'description=First Group Property' \
+--data-urlencode 'type=boolean'
+
+```
+{{/partial:tab}}
+{{partial:tab name="HTTP"}}
+```bash
+
+POST /api/2/taxonomy/group-property HTTP/1.1
+Host: amplitude.com
+Authorization: Basic MTIzNDU2NzgwMDoxMjM0NTY3MDA=
+
+Content-Length: 144
+
+group_type=Group%201&group_property=Group%20Property%201&type=boolean&description=First%20Group%20Property
+```
+{{/partial:tab}}
+{{/partial:tabs}}
+{{/partial:collapse}}
+
+#### Body parameters
+
+|<div class="big-column">Name</div>|Description|
+|-----|---------|
+|`group_property`| <span class="required">Required</span>. String. Name of the group property. <ul><li>Prefix custom group properties with `grp:`</li></ul>|
+|`group_type`|<span class="optional">Optional</span>. String. Name of the group type the group property belongs to. <ul><li>If the group type does not exist, we will return a `404 Not Found` error.</li><li>If the group property already exists on this group type, we will return a `409 Conflict` error.</li><li>If the group property already exists but not on this group type, we will create an override for this property.</li><li>If the group property does not exist anywhere, we will not create an override for this property.</li><li>If the group property exists and is an Amplitude sourced group property, providing any extra arguments other than `group_property` and `group_type` will result in an error because Amplitude sourced group properties cannot be edited.</li></ul>|
```

Comment:
```suggestion
|`group_type`|<span class="optional">Optional</span>. String. Name of the group type the group property belongs to. If the group type doesn't exist, Amplitude returns a `404 Not Found` error. If the group property already exists on this group type, Amplitude returns a `409 Conflict` error. If the group property already exists but not on this group type, Amplitude creates an override for this property. If the group property doesn't exist anywhere, Amplitude doesn't create an override for this property. If the group property exists and is an Amplitude-sourced group property, providing any extra arguments other than `group_property` and `group_type` results in an error because you can't edit Amplitude-sourced group properties. |
```